website/docs/docs/dbt-cloud-apis/sl-graphql.md

```diff
@@ -40,68 +48,323 @@ Authentication uses a dbt Cloud [service account tokens](/docs/dbt-cloud-apis/se
+  ) {
+    queryId
+  }
+}
+```
+
+**Query with a Where Filter** 
+
+The where filter takes a list argument (or a string for a single input). Depending on the object you are filtering on, there are a couple of parameters:
+ 
+ - `Dimension()` - This is used for any categorical or time dimensions. If used for a time dimension, granularity is required -  `Dimension('metric_time').grain('week')` or `Dimension('customer__country')`
+- `Entity()` - used for entities like primary and foreign keys - `Entity('order_id')`
+
+Note: If you prefer more strongly typed `where` clause, you can optionally use `TimeDimension()` to separate out categorical dimensions from time ones. The `TimeDimension` input takes the time dimension name and also requires granularity - an example is `TimeDimension('metric_time', 'MONTH')`.
+
+```graphql
+mutation {
+  createQuery(
+    environmentId: BigInt!
+    metrics:[{name: "order_total"}]
+    groupBy:[{name: "customer__customer_type"}, {name: "metric_time", grain: "month"}]
+    where:["{{ Dimension('customer__customer_type') }} = 'new'", "{{ Dimension('metric_time').grain('month') }} > '2022-10-01'"]
+    ) {
+     queryId
+    }
+}
 ```
 
+**Query with Order**
+```graphql
+mutation {
+  createQuery(
+    environmentId: BigInt!
+    metrics: [{name: "order_total"}]
+    groupBy: [{name: "metric_time", grain: "month"}] 
+    orderBy: [{metric: {name: "order_total"}}, {groupBy: {name: "metric_time", grain: "month"}, descending:true}]
+  ) {
+    queryId
+  }
+}
+```
+
+
+**Query with Limit**
+
+```graphql
+mutation {
+  createQuery(
+    environmentId: BigInt!
+    metrics: [{name:"food_order_amount"}, {name: "order_gross_profit"}]
+    groupBy: [{name:"metric_time, grain: "month"}, {name: "customer__customer_type"}]
+    limit: 10	
+  ) {
+    queryId
+  }
+}
+```
+
+**Query with Explain** 
+
+This takes the same inputs as the `createQuery` mutation.
+
+```graphql
+mutation {
+  compileSql(
+    environmentId: BigInt!
+    metrics: [{name:"food_order_amount"} {name:"order_gross_profit"}]
+    groupBy: [{name:"metric_time, grain:"month"}, {name:"customer__customer_type"}]
+  ) {
+    sql
+  }
+}
+```
+
+### Output Format & Pagination
+
+**Output Format**
+
+By default, output provided is in Arrow format. You can use the following parameter to convert to JSON. Due to performance limitations, we recommend the JSON parameter be only used for testing and validation. The JSON returned is a base64 encoded string. This means to access the JSON you must run it through a base64 decoder then you should receive the results in JSON. This is a JSON generated from pandas, this means you can either parse this back into a dataframe via pandas.read_json(json, orient="table") or just use the data directly with json["data"] and get the table schema with json["schema"]["fields"]
+
+
+```graphql
+{
+  query(environmentId: BigInt!, queryId: Int!, pageNum: Int {
```

Comment:
hey @WilliamDee @courtneyholocomb - should pageNum: Int include an exclamation?

Reply:
Nope it isn't required, it's defaulted to 1